docs/release-notes.md

```diff
@@ -1,5 +1,8 @@
 # Release Notes
 
+## Version 20.0 New Features
+:eight_spoked_asterisk: SMArt Email now supports MSAL for Office365/Outlook.com
```

Comment:
Can you add "Please check the configuration section for details on setup" or something so users know where to look when reading release notes?